src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -705,25 +705,25 @@ XamlObjectDefinition topLevelControl
 
 		private void BuildSingleTimeInitializer(IIndentedStringBuilder writer, string propertyType, string propertyName, Action propertyBodyBuilder, bool isStatic = true)
 		{
+			// The property type may be partially qualified, try resolving it through FindType
+			var propertySymbol = FindType(propertyType);
+			propertyType = propertySymbol?.GetFullName() ?? propertyType;
+
 			var sanitizedPropertyName = SanitizeResourceName(propertyName);
 
 			var propertyInitializedVariable = "_{0}Initialized".InvariantCultureFormat(sanitizedPropertyName);
 			var backingFieldVariable = "__{0}BackingField".InvariantCultureFormat(sanitizedPropertyName);
 			var staticModifier = isStatic ? "static" : "";
 			var publicPropertyType = FindType(propertyType)?.DeclaredAccessibility == Accessibility.Public
-				? GetGlobalizedTypeName(propertyType)
-				: "object";
-
-			// The property type may be partially qualified, try resolving it through FindType
-			var propertySymbol = FindType(propertyType);
-			propertyType = propertySymbol?.GetFullName() ?? propertyType;
+				? propertyType
+				: "System.Object";
```

Comment:
❗️ This `Scenario b`.
- Changed from `object` to `System.Object`

Reply:
This case is actually a fallback in case the type does not exist. In all strictness, it should fail.

This kind of code is dating back from the time where UWPSyncGenerator did not exist, and types may not exist, and this would avoid failing on too many errors during generation, and push those to the C# compilation step.

We can keep it that way for now, we'll revisit this when we'll tackle #119.

Reply:
🎗 Add a DataTemplate class that would conflict with the WinUI DataTemplate because of the current namespace. e.g. make an empty `DataTemplate` class in the `XamlGenerationTests` namespace (the default namespace for the current assembly, used by the generator)